tfx/orchestration/pipeline.py

```diff
@@ -189,3 +190,76 @@ def components(self, components: List[base_node.BaseNode]):
     # has all its dependencies visited.
     if len(self._components) < len(deduped_components):
       raise RuntimeError('There is a cycle in the pipeline')
+
+  def get_fuseable_components(self):
+    """
+       Returns a list of fuseable Beam component subraphs in sorted topological
+       order. For example:
+
+       [[component_a, component_b],[component_c, component_d, component_e]]
+
+       Experimental, backward compatability not guaranteed.
+    """
+
+    # Finds all Beam based components
+    beam_components = set()
+
+    for component in self.components:
+      if issubclass(component.EXECUTOR_SPEC.executor_class,
+                    base_executor.FuseableBeamExecutor):
+        beam_components.add(component)
```

Comment:
Optional: This can be
```
beam_components = set(component in self.components if issubclass(component.EXECUTOR_SPEC.executor_class, base_executor.FuseableBeamExecutor))
```

Reply:
I replaced `beam_components` with the function `_is_beam_component()`.